### PR TITLE
Listing page

### PR DESCRIPTION
**Pull request by git-0r**
# Created product listing page screens
ecom3
ecom4

### CODE REVIEW

**Review by Naman-Saxena1 — APPROVED** [review]
Code looks great, screen is also responsive 👍

File: css/listing_page.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+.top-navbar .center {
+  flex: 4;
+}
+
+.top-navbar .right {
+  flex: 2;
+}
+
+.top-navbar .search-input {
```
Comment (varun1233) [inline css/listing_page.css:24]: Suggestion : Avoid css chaining if possible. It can lead to specificity issues.

File: css/listing_page.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+}
+
+.product-listing-navigation {
+  width: 100%;
+  padding: 2rem;
+}
+
+.banner {
+  width: 100%;
+  height: 400px;
```
Comment (varun1233) [inline css/listing_page.css:59]: use rem

File: index.css [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  width: 100%;
+  padding: 1rem;
+  margin-bottom: 1rem;
+  border-radius: var(--border-radius-sm);
+  background-color: var(--clr-primary-outline);
+}
+
+.banner {
+  width: 100%;
+  height: 400px;
```
Comment (varun1233) [inline index.css:40]: use rem

File: index.html [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+        </div>
+    </nav>
+
+    <div class="offer-notification text-align-center">Get Free Shipping On Orders Above &#8377;999</div>
+    <!-- BANNER -->
+
+    <div class="banner">
+        <img class="banner-image"
+            src="https://res.cloudinary.com/clouduser/image/upload/w_1200,h_500,c_fill/v1645362667/download_d2hg44.jpg"
+            alt="">
```
Comment (varun1233) [inline index.html:55]: Suggestion: It's a good practice to provide alt attribute for images

**Review by Sahil25061999 — APPROVED** [review]
Same suggestion as varun use rem for values greater than 10. Add hover effects to card buttons and the nav icon buttons.

**Pull request merged**